src/lib/realmdb/feedSource/NewsSource.js

```diff
@@ -1,4 +1,6 @@
 import moment from 'moment'
+import Mutex from 'await-mutex'
```

Comment:
revert this. should be wrapped to mutex in realmdb class 
https://github.com/GoodDollar/GoodDAPP/blob/news-sync-mutex/src/lib/realmdb/RealmDB.js#L198